sdk/servicebus/azure-servicebus/tests/test_queues.py

```diff
@@ -863,10 +865,10 @@ def test_queue_by_queue_client_conn_str_receive_handler_with_autolockrenew(self,
                     messages.append(message)
                     assert not message.expired
                     renewer.register(message, timeout=60)
-                    print("Registered lock renew thread", message.locked_until, datetime.now())
                     time.sleep(50)
                     print("Finished first sleep", message.locked_until)
                     assert not message.expired
+                    time.sleep((message.locked_until - datetime.now()).total_seconds() + 1)
                     time.sleep(25)
```

Comment:
I guess `time.sleep(25)` can be removed?

Reply:
Same as below, needed for timing out the autorenewer and then the message, I've added comments.